turnerlabs/yt_push
# DRACO-3739 Add a Drush command that quickly adds a typical media type

To test:

* Enable the module
* Run `drush config-add-youtube`
* Check the resulting media type against the acceptance criteria in the ticket.
* Check that you can finish the configuration on the new media type and upload a video to YouTube.
* Check that you can re-run the command and it'll create a new media type.
